UIKit/Tasks/NSLayoutAnchorViewController.swift

```diff
@@ -0,0 +1,83 @@
+// NSLayoutAnchorViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран создание светофора с помощью Якорей
+final class NSLayoutAnchorViewController: UIViewController {
+    // MARK: - Visual Components
+
+    private let redView = UIView()
+    private let yellowView = UIView()
+    private let greenView = UIView()
+    private let grayView = UIView()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupLayoutAnchor()
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        redView.layer.cornerRadius = redView.bounds.height / 2
+        yellowView.layer.cornerRadius = yellowView.bounds.height / 2
+        greenView.layer.cornerRadius = greenView.bounds.height / 2
+    }
+
+    // MARK: - Private Methods
+
+    private func setupLayoutAnchor() {
+        setupyellowViewAnchor()
+        setupRedViewAnchor()
+        setupGreenViewAnchor()
+        setupGrayViewAnchor()
+    }
+
+    private func setupUI() {
+        redView.backgroundColor = .red
+        yellowView.backgroundColor = .yellow
+        greenView.backgroundColor = .green
+        grayView.backgroundColor = .gray
+
+        for item in [grayView, redView, yellowView, greenView] {
+            view.addSubview(item)
+            item.translatesAutoresizingMaskIntoConstraints = false
+        }
+    }
+
+    private func setupyellowViewAnchor() {
```

Comment:
```suggestion
    private func setupYellowViewAnchors() {
```
Ну и везде по коду Anchors по идее